source/delegate/package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "@airswap/order-utils": "0.3.16",
     "@airswap/test-utils": "0.1.3",
+    "@airswap/transfers": "0.1.0",
```

Comment:
is it on rinkeby? should it be 0.0.1 instead?

Reply:
replacing it down, it shouldn't be updated.